lib/oli/delivery/metrics.ex

```diff
@@ -1117,13 +1117,14 @@ defmodule Oli.Delivery.Metrics do
             "CAST(? as float) / NULLIF(CAST(? as float), 0.0)",
             summary.num_first_attempts_correct,
             summary.num_first_attempts
-          )
+          ),
+          summary.num_first_attempts
```

Comment:
v2 analytics now return the `summary.num_first_attempts` as the 3rd element in the tuple